ocaml/quicktest/jbuild

```diff
@@ -31,8 +31,8 @@ let () = Printf.ksprintf Jbuild_plugin.V1.send {|
   (public_name quicktestbin)
   (package xapi)
   (flags (:standard -bin-annot %s -warn-error +a-3-4-6-9-27-28-29-52))
-  (libraries (
-   xapi_internal
-  ))
+  (libraries
+    (alcotest
```

Comment:
You also use xapi-stdext-pervasives below and unix. I would also double check rpclib and threads